### PR TITLE
fix Issue 23566 - ImportC: __PRETTY_FUNCTION__ is not defined

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

**Comment by WalterBright** [comment]
The Azure failure is:
```
>>> TARGET FAILED: runnable\test23566.c
FAILED targets:
- runnable\test23566.c
##[error]Cmd.exe exited with code '1'.
```
No other information given.

File: compiler/test/runnable/test23566.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+file: main
+test: test
+---
+*/
+
+int printf(const char *, ...);
+
+void test()
+{
+//    printf("test: %s\n", __PRETTY_FUNCTION__);
```
Comment (ljmf00) [inline compiler/test/runnable/test23566.c:12]: Why this is commented out?

**Comment by WalterBright** [comment]
```
src\importc.h(82) : Preprocessor error: macro '__PRETTY_FUNCTION__' can't be #undef'd or #define'd

```
Sigh. Microsoft's preprocessor defeats the purpose of a preprocessor.

**Comment by WalterBright** [comment]
Found an easier way.

File: compiler/test/runnable/test23566.c (hunk 1 above)
Comment (WalterBright) [reply compiler/test/runnable/test23566.c]: removed that file

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**